playbooks/roles/jenkins/templates/aws_config.j2

```diff
@@ -1,7 +1,13 @@
-{% for role in assume_roles %}
+{% for role in assume_cloudtrail_roles %}
 
 [profile {{role.profile.name}}]
 credential_source=Ec2InstanceMetadata
 region={{role.profile.region}}
 role_arn={{role.profile.role_arn}}
 {% endfor %}
+{% for role in aws_infrastructure_roles %}
+
+[profile {{role.profile.name}}]
+credential_source=Ec2InstanceMetadata
+role_arn={{role.profile.role_arn}}
```

Comment:
Why isn't the region required for these? 🤔

Reply:
I was hoping someone else could answer that 😹  I'm not 100% sure if it's needed for the Cloudtrail profiles!

Reply:
In practice though, the Jenkins user is only going to be using `aws s3` commands, so they should be region agnostic.

Reply:
If we can take the regions out for the cloudtrail profiles as well that would be great!

Reply:
Maybe what we can do is add

```
[default]
region=eu-west-1
```

to the top of the file, then we don't need regions for the individual profiles, and we can use one for loop for all of the profiles.

Reply:
Yeah, I think adding a default region is a good shout.

I think I'd rather keep the loops separate though as they're coming from different creds variables?

Reply:
But then we'd have to add new loops every time we want to add more creds variables? I guess that's not a big deal really

Reply:
Given the internal structure of the vars is not very visible, I think some verbosity here is fine.

Reply:
I've pushed a commit to add the default region. However as Cloudtrail is region-specific I'm going to keep the explicit setting for each profile there for now.